package.json

```diff
@@ -11,13 +11,21 @@
     "xtend": "^4.0.1"
   },
   "devDependencies": {
-    "standard": "*"
+    "nan": "^2.5.1",
+    "standard": "*",
```

Comment:
could you pin this to latest standard? (i should have done that from the beginning)